app/src/main/java/com/brainwallet/di/Module.kt

```diff
@@ -97,16 +97,17 @@ private fun provideOkHttpClient(): OkHttpClient = OkHttpClient.Builder()
     .addInterceptor { chain ->
         val request = chain.request()
         runCatching {
-            val result = chain.proceed(request)
-            if (result.isSuccessful.not()) {
-                throw HttpException(
-                    retrofit2.Response.error<Any>(
-                        result.code,
-                        result.body ?: result.peekBody(Long.MAX_VALUE)
+            chain.proceed(request).use { response ->
```

Comment:
use `.use{ }` that actually implement closable

Reply:
Ok. coll. lets check it out